templates/task_config_prepro.yaml

```diff
@@ -0,0 +1,6 @@
+VM_PARAMS: ONCE
+VM_GEN: ONCE
+# if you select the VM_PERT option you need to re-install (install_cybershake.py) after generating perturbations to
+# enable the perturbation to take effect
+VM_PERT: NONE # ALL
+INSTALL_FAULT: ONCE
```

Comment:
You didn't add the "no_vm_pert" step? I would have expected that to be easy enough to add, and give install the dual dependency